api/src/main/java/org/openmrs/module/adminui/account/Account.java

```diff
@@ -130,7 +131,30 @@ public void setGender(String gender) {
 	public String getGender() {
 		return getPerson().getGender();
 	}
-	
+
+	public List<PersonAttribute> getActiveAttributes() {
+		return getPerson().getActiveAttributes();
+	}
+
+	/**
+	 * Gets the person attribute value for the specified person for the getPersonAttributeTypeByUuid
+	 * that matches the specified uuid
+	 *
+	 * @return the attribute value
+	 */
+	public PersonAttribute getPersonAttribute(String attributeTypeUuid) {
+		Person person = getPerson();
+		if (person != null) {
+			PersonAttribute attr = person.getAttribute(Context.getPersonService().getPersonAttributeTypeByUuid(
+					attributeTypeUuid));
+			if (attr != null) {
```

Comment:
What value does checking for not null add?

Reply:
No need it here, removed